migtests/tests/pg-tests/pg-views/env.sh

```diff
@@ -1,3 +1,4 @@
 export SOURCE_DB_TYPE="postgresql"
 export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"pg_views"}
-export SOURCE_DB_SCHEMA="public"
+export SOURCE_DB_SCHEMA="test_case"
+export TARGET_DB_SCHEMA="TEST_Case"
```

Comment:
this is a special case, don't remove `public` schema case. We couple of cases for tests.
1. public schema(i.e. default)
2. non-public schema name
3. case sensitive non-public schema name

Reply:
yes for target-db-schema

Reply:
Okay, added a case for 2nd one as 1st and 3rd are already added.